onboarding_auth/onboardingAuthPresentation/src/main/java/com/alexafit/onboardingAuthPresentation/login/LoginScreen.kt

```diff
@@ -55,6 +60,8 @@ fun LoginScreen(
     val spacing = LocalSpacing.current
     val context = LocalContext.current
     val keyboardController = LocalSoftwareKeyboardController.current
+    var isLoading by remember { mutableStateOf(false) }
```

Comment:
I like to have 1 state class for each screen, and have all that state there

Ex: LoginState